### PR TITLE
fix(kitty): don't use IFD

### PR DESCRIPTION
**Pull request by lovesegfault**
https://github.com/danth/stylix/issues/159

### CODE REVIEW

File: modules/kitty/hm.nix [hunk 1]
```diff
@@ -32,7 +32,10 @@ in {
         size = config.stylix.fonts.sizes.terminal;
       };
       settings.background_opacity = with config.stylix.opacity; "${builtins.toString terminal}";
-      extraConfig = builtins.readFile theme;
+      extraConfig = ''
+        include stylix.conf
```
Comment (danth) [inline modules/kitty/hm.nix:36]: Could this work without linking another file in the home directory?
```suggestion
        include ${theme}
```

Comment (lovesegfault) [reply modules/kitty/hm.nix:36]: Probably, let me give it a shot

Comment (lovesegfault) [reply modules/kitty/hm.nix]: Yup! Done :)

**Review by danth — APPROVED** [review]
Great, thanks!

**Pull request merged**